.circleci/config.yml

```diff
@@ -13,16 +23,17 @@ jobs:
       - run:
           name: Download dependencies
           command: lein deps
+      - run:
+          name: Eastwood
+          command: lein eastwood
       - run:
           name: Run the tests
-          command: lein trampoline test
+          command: lein test-ci
       - run:
           name: Code Coverage
-          command: lein trampoline cloverage --codecov
-      - run:
-          name: Upload Coverage
-          command: bash <(curl -s https://codecov.io/bash) -f target/coverage/codecov.json
-          when: always
+          command: lein cloverage --codecov
+      - codecov/upload:
+          file: target/coverage/codecov.json
```

Comment:
@neeasade this comment got resolved but not addressed?

Reply:
my bad (slow down) --> https://github.com/circleci/bond/pull/57